src/player.cpp

```diff
@@ -12942,6 +12976,23 @@ int player::get_hp_max( hp_part bp ) const
     return hp_total;
 }
 
+int player::get_stamina_max()
+{
+    return 1000;
```

Comment:
A STA (Stamina) stat wouldn't so bad I think, to differentiate from STR (the pure brute force you can apply) and DEX (how dextrous you are). Though that would require a complex rebalance of many things...